README.md

```diff
@@ -2,10 +2,24 @@
 
 ![Experimental](https://img.shields.io/badge/experimental-important.svg?style=for-the-badge)
 
-NOTICE: this library is still not published to package managers. Stay tuned.
-
 Deploy CDK applications through GitHub workflows.
```

Comment:
Maybe add a pointer here to CDK Pipelines as well, and say "THIS is like THAT, except..." ?

Reply:
Also like two more lines about the purpose would be useful. Your target audience is someone with passable knowledge about computers, software systems, GitHub and AWS, but not necessarily all the intricacies of deploying, how to setup their CDK app, and what other solutions are available on the market.

Reply:
I've reworked this section, let me know if more detail is needed. The CDK Pipeline module starts off the same way, so I thought I'd pay homage